scripts/checkconfig.py

```diff
@@ -62,7 +62,7 @@
 
 def separate_location_lines(dirs):
     for dir in dirs:
-        print "    ", dir
+        print ("    ", dir)
```

Comment:
Please remove space between "print" and "(", print is a normal function in Python3, and should be used as such (see e.g. 2 lines above, `separate_location_lines(dirs)`)